egui/src/widgets/plot/mod.rs

```diff
@@ -115,6 +121,70 @@ impl PlotMemory {
 
 // ----------------------------------------------------------------------------
 
+/// Indicates a vertical or horizontal cursor line in plot coordinates.
+#[derive(Copy, Clone, PartialEq)]
+enum Cursor {
+    Horizontal { y: f64 },
+    Vertical { x: f64 },
+}
+
+/// Contains the cursors drawn for a specific frame of a plot.
```

Comment:
`frame` as in "frames per second"? Or just is it just "for a specific plot widget"? (each widget has a unique `id`)

Reply:
It's what was drawn for a specific plot widget in a frame with the "frames per second" meaning.